addons/account/models/account_payment_term.py

```diff
@@ -15,15 +15,21 @@ class AccountPaymentTerm(models.Model):
     def _default_line_ids(self):
         return [Command.create({'value': 'balance', 'value_amount': 0.0, 'days': 0, 'end_month': False})]
 
+    def _default_example_amount(self):
+        return self._context.get('amount_total') or 100  # Set 100 as default value for example amount if either no amount_total is given in context or if the given amount in the context is False
```

Comment:
- This will also erase `0`
- The code is quite clear, do you think the comment is really needed?

Reply:
- Yup it's wanted as an example with 0 wouldn't be interesting to show the distribution of the payment terms
- Asked by wan to explain why we didn't pass the example amount/date directly in the xml file (i.e. avoid using `default_example_amount` in the xml which would override the `example_amount` here even if it's `False`)

Reply:
I would still use `default_amount_total` and `default_invoice_date`, this way the hack is more obvious and the comment can be shorter:
> `force default value if context set to void`

Reply:
I mean `default_example_amount` and `default_example_date` :man_facepalming:

Reply:
If we use `default_example_amount`, the ORM will immediately use that value (which could be `False`) without going in this function, so another name has to be used...

Reply:
So perhaps we should not prepend `default_` at all ?

Reply:
Oh, right
![:facepalm:](https://vignette.wikia.nocookie.net/animaljam/images/0/0f/FACEPALM.gif/revision/latest)

Well `example_amount` then, at least the intention of the context stays clear